pack/set.go

```diff
@@ -91,6 +92,21 @@ func NewSet(db string) (*Set, error) {
 	return NewSetPacks(packs...), nil
 }
 
+// globEscapes uses these escapes because filepath.Glob does not understand
+// backslash escapes on Windows.
+var globEscapes = map[string]string{
+	"*": "[*]",
+	"?": "[?]",
+	"[": "[[]",
+}
+
+func escapeGlobPattern(s string) string {
+	for char, escape := range globEscapes {
+		s = strings.Replace(s, char, escape, -1)
```

Comment:
Nit: in newer versions of Go, I believe that `strings.ReplaceAll` is preferred.

Reply:
Yes, but I think this repository still uses Travis, which uses Go 1.11.  `strings.ReplaceAll` is new in 1.12.  I've been bitten by this in the past (multiple times, even).